tests/dummy/app/routes/nodes/detail.js

```diff
@@ -130,8 +131,28 @@ export default Ember.Route.extend({
             } else {
                 console.log('You do not have permissions to delete this node');
             }
+        },
+        addNodeLink(targetNodeId) {
+            var node = this.modelFor(this.routeName);
+            if (node.get('currentUserPermissions').indexOf('write') !== -1) {
```

Comment:
It would be good to use the permissions constants for all of these permissions strings.